src/Progress/LinearProgress.js

```diff
@@ -8,38 +8,56 @@ import withStyles from '../styles/withStyles';
 
 const transitionDuration = 4; // 400ms
 
+const getBar = color => ({
```

Comment:
Should like we have duplicated style injected in the page. What about sharing what's common between the colors?